mos_tests/object_storage/test_crud_operations.py

```diff
@@ -0,0 +1,450 @@
+        s3cmd_client.bucket_make(self.BUCKETNAME)
+        s3cmd_client.bucket_put_file(self.BUCKETNAME, f_path)
+        s3cmd_client.bucket_del_file(self.BUCKETNAME, f_name)
+        ls_in = s3cmd_client.bucket_ls(self.BUCKETNAME)
+        s3cmd_client.bucket_remove(self.BUCKETNAME, recursive=True)
+        obj_path = '{0}/{1}'.format(self.BUCKETNAME, f_name)
+        assert obj_path not in ''.join(ls_in)
+
+    @pytest.mark.testrail_id('842479')
+    def test_object_storage_s3cmd_obj_upload_big(
+            self, s3cmd_client, ctrl_remote):
+        """Upload big object to Object Storage (RadosGW)
+
+        Actions:
+        1. Create file (object) on controller.
+        2. Create container via s3cmd cli utility.
+        3. Put object to container with option 'multipart-chunk-size-mb'.
+        4. With list check that object present in container.
+        5. Check that size of container is the same as it was created.
+        6. Check expected and actual chunk slice number.
+        5. Delete container.
+        """
+        file_size_b = str(self.f_size_big * 1024 * 1024)
+        f_path, f_name = self.create_file_on_node(
+            ctrl_remote, self.f_size_big)
+        s3cmd_client.bucket_make(self.BUCKETNAME)
+        put_out = s3cmd_client.bucket_put_file(
+            self.BUCKETNAME, f_path, chunk=self.chunk_size)
+        ls_in = s3cmd_client.bucket_ls(self.BUCKETNAME)
+        s3cmd_client.bucket_remove(self.BUCKETNAME, recursive=True)
+        obj_path = '{0}/{1}'.format(self.BUCKETNAME, f_name)
+        # check object present in ls
+        assert obj_path in ''.join(ls_in)
+        # check size of obj is correct
+        assert file_size_b in ''.join(ls_in)
+        # check num of slices is correct
+        exp_chunk_slices = math.ceil(
+            float(self.f_size_big) / float(self.chunk_size))
+        # MBs left in last slice
+        mbleft = int(
+            self.f_size_big - (self.chunk_size * (exp_chunk_slices - 1)))
+        exp_string = (
+            "upload:.+{f_path}.+->.+s3://{obj_path}.+"
+            "[part {part} of {part}, {mbleft}MB]"
+        ).format(f_path=f_path, obj_path=obj_path, part=int(exp_chunk_slices),
+                 mbleft=mbleft)
+        # check string present in output
+        assert re.match(exp_string, put_out) is not None
+        # upload: '/root/s3cmd_test_obj' ->
+        # 's3://TESTBUCKET1772/s3cmd_test_obj'  [part 4 of 4, 138MB] [1 of 1]
+
+
+@pytest.mark.undestructive
+@pytest.mark.check_env_('is_ceph_enabled')
+@pytest.mark.check_env_('is_radosgw_enabled')
+class TestObjectStorageSWIFT(TestBase, Helpers):
+    """Object Storage - CRUD Operations Tests
+    """
+    BUCKETNAME = 'TESTBUCKET{0}'.format(random.randint(0, 10000))
+    f_size = 100         # MB
+    f_size_big = 5432    # MB
+    segment_size = 1024  # MB
+
+    @classmethod
+    @pytest.fixture(scope='class', autouse=True)
+    def prepare_controller_for_swift(cls, env, ctrl_remote):
```

Comment:
With pytest we can have more that one fixture, and i believe, that each name should be linked to it't role to preparation or finalization of tests.